app/controllers/application_controller.rb

```diff
@@ -2,6 +2,7 @@
 
 class ApplicationController < ActionController::Base
   include Pagy::Backend
+  include CurrentCart
```

Comment:
now that you set it here you can remove it from the other controllers

Reply:
They've already been deleted.